src/test/java/Homework23.java

```diff
@@ -0,0 +1,62 @@
+import PageObjectModel.HomePage;
+import PageObjectModel.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework23 extends BaseTest {
+
+    /*@Test
+    public void deleteAPlayList() throws InterruptedException{
+        //Navigate to "https://qa.koel.app/".
+        launchBrowser(url);
+
+        // Log in with your credentials.
+        navigateToLoginPage();
+        provideEmail("demo@class.com");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        Thread.sleep(2000);
+
+        //Steps to delete a Playlist
+        openPlaylist();
+        deletePlaylist();
+        Assert.assertTrue(playListDeleted());
+    }*/
+
+    @Test
+    public void deleteAPlayList() throws InterruptedException{
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.